### PR TITLE
Remove unused saver for 2d ldm bundle

### PR DESCRIPTION
**Pull request by Can-Zhao**
Fixes # .

### Description
Remove unused saver for 2d ldm bundle

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Review by yiheng-wang-nv — APPROVED** [review]
confirmed that removing "saver" will not impact on the inference result.

File: models/brats_mri_axial_slices_generative_diffusion/configs/inference.json [hunk 1]
```diff
@@ -93,12 +93,6 @@
         "_requires_": "@set_timesteps"
     },
     "sample": "$@inferer.sampling_fn(@noise, @autoencoder, @diffusion, @noise_scheduler)",
-    "saver": {
-        "_target_": "SaveImage",
-        "_requires_": "@create_output_dir",
-        "output_dir": "@output_dir",
-        "output_postfix": "@output_postfix"
-    },
     "generated_image": "$@sample",
```
Comment (yiheng-wang-nv) [inline models/brats_mri_axial_slices_generative_diffusion/configs/inference.json:96]: could you please also update the version and changelog in metadata?

Comment (Can-Zhao) [reply models/brats_mri_axial_slices_generative_diffusion/configs/inference.json:96]: Done! Thanks

**Comment by Can-Zhao** [comment]
> confirmed that removing "saver" will not impact on the inference result.

Yes, tested

**Pull request merged**